autogen/main/outputs.tf.tmpl

```diff
@@ -128,18 +133,13 @@ output "release_channel" {
   value       = var.release_channel
 }
 
-output "identity_namespace" {
-  description = "Workload Identity namespace"
-  value       = length(local.cluster_workload_identity_config) > 0 ? local.cluster_workload_identity_config[0].identity_namespace : null
+output "workload_pool" {
```

Comment:
I don't think there's a real need to change this output name, since it's still pointing to the same value.